cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -56,6 +55,7 @@ pub struct CoreEnvironment<State = ReadOnly> {
     ///
     /// Commonly /.../.flox/env/
     env_dir: PathBuf,
+    include_fetcher: IncludeFetcher,
```

Comment:
comment as to what this is for and that the path it contains is separate from `env_dir`

Reply:
Fixed